app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
 
     defaultConfig {
         applicationId "fr.gaulupeau.apps.InThePoche"
-        minSdkVersion 11
+        minSdkVersion 19
```

Comment:
so that we can use getExternalFilesDirs() to determine the real SD card and not only internal emulated external storage: https://developer.android.com/reference/android/content/Context.html#getExternalFilesDirs(java.lang.String)

if you have an other way to determine the REAL sd card, i would be happy to see a commit :)